playground/backend/internal/cache/local_cache.go

```diff
@@ -0,0 +1,112 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"time"
+)
+
+const cleanupInterval = 5 * time.Second
+
+type LocalCache struct {
+	cleanupInterval time.Duration
+	items           map[uuid.UUID]map[Tag]Item
+}
+
+type Item struct {
+	value      interface{}
+	created    time.Time
+	expiration int64
+}
+
+func newLocalCache() *LocalCache {
+	items := make(map[uuid.UUID]map[Tag]Item)
+	ls := &LocalCache{
+		cleanupInterval: cleanupInterval,
+		items:           items,
+	}
+
+	go ls.startGC()
+	return ls
+
+}
+
+func (ls *LocalCache) Get(pipelineId uuid.UUID, tag Tag) (interface{}, error) {
+	item, found := ls.items[pipelineId][tag]
+	if !found {
+		return nil, fmt.Errorf("Item with pipelineId: %s and tag: %s not found.", pipelineId, tag)
```

Comment:
does it make sense to has common errors for interface? I mean does Redis implementation has a similar error? if yes - we can create a Golang error for these purposes.